src/main/resources/application.properties

```diff
@@ -73,3 +73,6 @@ email.subscription.daily.cron=0 0 2 * * ?
 policynotifications/mp-rest/url=http://localhost:8090
 policynotifications/mp-rest/connectTimeout=2000
 policynotifications/mp-rest/readTimeout=2000
+
+# qute
+quarkus.qute.property-not-found-strategy=throw-exception
```

Comment:
If an exception is thrown while an e-mail is being sent, what will be the consequence for the user? Will there be another e-mail attempt later or will the e-mail be lost forever?

I suppose we would catch that kind of exceptions before it goes to production so that's probably a very low risk, but since it's still there I want to make sure of what happens after the exception.

Reply:
After further reading of the code, it doesn't look like the e-mail would be canceled in case of exception.

Reply:
If an exception is throw there, the template or the data is probably wrong, nothing we can do in that case. There could be some enhancements to report this broken request to the user/app.